docs/docs/getting-started/installation.md

```diff
@@ -76,6 +76,19 @@ bastionlab_server.stop(srv)
 docker run -p 50056:50056 -d mithrilsecuritysas/bastionlab:latest
 ```
 
+### Configurating the Docker image
+If you want to modify the server's configuration, you should rather run the container like so:
+```
+docker create -p 50056:50056 --name bastionlab-srv mithrilsecuritysas/bastionlab:latest
+docker cp <your_updated_config>.toml bastionlab-srv:/app/bin
+docker start bastionlab-srv
+```
+
+Here is an example of reasonable config.toml:

```

Comment:
Here, also replace by this:
_Here is an example of a reasonable `config.toml`:_

Reply:
What does reasonable mean here?

Reply:
_By default, here's the `config.toml`:_